typescript/src/lib/SkyraClient.ts

```diff
@@ -60,7 +60,7 @@ export class SkyraClient extends SapphireClient {
 	public readonly analytics: AnalyticsData | null;
 
 	@enumerable(false)
-	public readonly guildMemberFetchQueue: GuildMemberFetchQueue = new GuildMemberFetchQueue(this);
+	public readonly guildMemberFetchQueue: GuildMemberFetchQueue = new GuildMemberFetchQueue();
```

Comment:
is this a bugfix 🤔 ?

Reply:
Nah, we already have access to `client` via container.